src/person.rb

```diff
@@ -1,9 +1,11 @@
 require_relative './nameable'
+require_relative './capitalize_decorator'
+require_relative './trimmer_decorator'
 
 # Make sure that your Person class inherits from Nameable
 class Person < Nameable
-  attr_accessor :name, :age, :parent_permission
-  attr_reader :id
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ] Again, `rental` here should be marked with `attr_accessor`, as we intend to both _access (get)_ as well as _modify (set)_ it. Kindly ensure to improve upon this aspect.

Reply:
Change implement

Reply:
@Recillah-Khamala 
Kindly resubmit your project for a review

Cheers! 😃

Reply:
I have already submitted

Reply:
Hi @Recillah-Khamala 

yes, I can see you have the [Key Features](https://github.com/Recillah-Khamala/School-Library/tree/associations#key-features) sub-section, but not the `## Future Features`.
![Screenshot from 2022-12-23 04-49-33](https://user-images.githubusercontent.com/83042742/209261308-699be145-5af3-4540-aaba-d79c64b5b522.png)

Thanks